model_analyzer/reports/report_manager.py

```diff
@@ -379,6 +380,14 @@ def _build_summary_report(self, report_key, num_configs, statistics):
         if self._result_manager._profiling_models_concurrently():
             caption_results_table = caption_results_table + " Per model values are parenthetical."
 
+        if run_config.model_run_configs()[0].is_ensemble_model():
```

Comment:
Yes, it's because we're not supporting ensemble in a multi-model environment

Reply:
seems like this would be really messy to clean up in the future. Can we at least limit it to one reference to [0]? AKA put model_run_configs[0] in a well named variable and use that?